scripts/preinstall.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
+
+var lsp_server_version = "1.0.0-SNAPSHOT";
```

Comment:
what is the policy to clear the snapshots after a release on oss.sonatype? How much time do we have to update it?

Reply:
policy is not very clear but the artifact is downloaded at build time so there is no problem (I thought it was  during installation but Lars checked that it is not, thanks!). The build will break but no user will be impacted